mythril/interfaces/cli.py

```diff
@@ -556,6 +569,17 @@ def execute_command(
             print("Disassembly: \n" + disassembler.contracts[0].get_creation_easm())
 
     elif args.command in ANALYZE_LIST:
+
+        if args.func_whitelist:
+            _whitelist = args.func_whitelist.split(",")
```

Comment:
There can be multiple args, so we can't split by comma. `batchTransfer(uint a, uint b)` function will be split